pkg/agent/controller/networkpolicy/fqdn.go

```diff
@@ -405,80 +417,111 @@ func (f *fqdnController) deleteRuleSelectedPods(ruleID string) error {
 
 func (f *fqdnController) onDNSResponse(
 	fqdn string,
-	responseIPs map[string]net.IP,
-	lowestTTL uint32,
+	newIPWithExpiration map[string]ipWithExpiration,
```

Comment:
s/newIPWithExpiration/newIPsWithExpiration

there are potentially multiple IPs because it's a map